commit d76389b4d1832660192dff54e8d43a341ca4fd98
Merge: 4611759 3613078
Author: Scott Lahteine <thinkyhead@users.noreply.github.com>
Date:   Tue Jul 19 12:43:50 2016 -0700

    Merge pull request #4340 from esenapaj/Follow-up-the-PR-#4335
    
    Follow-up the PR #4335 (Debug char, fix compile errors for lcd pins)

diff --git a/Marlin/ultralcd.cpp b/Marlin/ultralcd.cpp
index f96d74c..f1f71b3 100755
--- a/Marlin/ultralcd.cpp
+++ b/Marlin/ultralcd.cpp
@@ -2741,7 +2741,11 @@ void lcd_update() {
 void set_utf_strlen(char* s, uint8_t n) {
   uint8_t i = 0, j = 0;
   while (s[i] && (j < n)) {
-    if ((s[i] & 0xC0u) != 0x80u) j++;
+    #if ENABLED(MAPPER_NON)
+      j++;
+    #else
+      if ((s[i] & 0xC0u) != 0x80u) j++;
+    #endif
     i++;
   }
   while (j++ < n) s[i++] = ' ';
